src/makeCalculator.js

```diff
@@ -37,7 +37,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: (number) => {
+      calculator.result += number;
+    },
+    subtract: (number) => {
+      calculator.result -= number;
+    },
+    divide: (number) => {
+      calculator.result /= number;
+    },
+    multiply: (number) => {
+      calculator.result *= number;
+    },
+    operate: (callback, number) => {
+      callback(number);
```

Comment:
pass result and incoming number here, don't manipulate with result in callbacks

Reply:
I don't understand what is problem?

Reply:
pass `this.result` as an argument to callback, in callbacks just return result